### ISSUE TITLE
Regression for sub projects while updating to Play 2.7

### ISSUE THREAD

**Issue by LeComptoirDesPharmacies** [playframework/play-ebean#209]
Hi,

As described in the following issue (https://github.com/playframework/playframework/issues/10619) on Play Framework project, there is a regression with Play Ebean plugin and sub projects.

It seems to come from this PR where configuration loading changed from Configuration to Config.
https://github.com/playframework/play-ebean/pull/157

However, the behavior between both code is not the same. Is it intended ?

Yours faithfully,
LCDP

**Comment by LeComptoirDesPharmacies** [comment]
Hi,

Any news about this pull request ?
Do you think that it could be merged in the next release (v6.1.0) ?

Yours faithfully,
LCDP

**Comment by jmswenski** [comment]
Hey @LeComptoirDesPharmacies, I merged this PR after testing it successfully on a play 2.8.x project and since we now have Sonatype builds working thanks to @octonato and @felipebonezi there is a snapshot build up in Sonatype with this fix. You can try it out like this:

```
resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
addSbtPlugin("com.typesafe.sbt" % "sbt-play-ebean" % "6.1.0-RC3+3-36c6f650-SNAPSHOT")
```

I'm working on a couple more things and have a few questions to resolve before I make an official 6.1.0 version. But give the snapshot a try!

**Comment by mkurz** [comment]
Should be fixed by #210
